src/formats/notion-api/database-helpers.ts

```diff
@@ -731,6 +618,146 @@ function sanitizePropertyKey(key: string): string {
 	return key;
 }
 
+/**
+ * Extract property types that should be written to types.json
+ * Maps all Notion property types to Obsidian property types following the same logic as mapDatabaseProperties
+ * Returns a mapping of property name -> Obsidian type
+ */
+function extractPropertyTypesForTypesJson(
+	dataSourceProperties: any,
```

Comment:
Please don't use any. This can be `Record<string, any>`. Remove the `as Record<string, any>` from the for loop below.